src/features/user/CompleteSignup/index.tsx

```diff
@@ -255,380 +255,378 @@ export default function CompleteSignup(): ReactElement | null {
   if (contextLoaded && token && user === null) {
     return (
       <div
-        className={styles.signupPage}
+        className={styles.completeSignup}
         style={{
           backgroundImage: `url(${process.env.CDN_URL}/media/images/app/bg_layer.jpg)`,
         }}
       >
-        <div className={styles.signupContainer}>
-          <div
-            className={requestSent ? styles.signupRequestSent : styles.signup}
-            style={{
-              backgroundColor:
-                theme === 'theme-light'
-                  ? themeProperties.designSystem.colors.white
-                  : themeProperties.dark.backgroundColor,
-              color:
-                theme === 'theme-light'
-                  ? themeProperties.designSystem.colors.coreText
-                  : themeProperties.dark.primaryFontColor,
-            }}
+        <div
+          className={
+            isProcessing ? styles.signupFormOverlay : styles.signupFormBase
```

Comment:
Functionally, this is ok. But the class names are confusing.

I'd suggest leaving `.signupFormContainer`, and using `.signupFormContainer.processing` in tandem to disable form submission and show processing state.